ui/app/store/actionConstants.test.js

```diff
@@ -0,0 +1,82 @@
+import assert from 'assert';
+import freeze from 'deep-freeze-strict';
+import reducers from '../ducks';
```

Comment:
This seems to be testing the reducers more than the action constants :thinking: 

This is definitely a huge improvement from how these were organized before though!